src/main/java/io/anserini/collection/Iso19115Collection.java

```diff
@@ -132,8 +164,8 @@ public Document(JsonNode json) {
               .get("gmd:geographicElement").get("gmd:EX_GeographicBoundingBox").get("gmd:southBoundLatitude").get("gco:Decimal").asDouble();
       // ensuring that a single coordinate location will be drawn as a small rectangle
       if (latitude[0] == latitude[2]) {
-        latitude[0] -= 0.00001;
-        latitude[2] += 0.00001;
+        latitude[0] -= 0.01;
```

Comment:
I think this should be done in the UI - I don't think we should be altering the records?

Reply:
because we are putting longitude/latitude in a string, it would be a lot more painful to do this on the frontend, I don't think we are altering the records in this case, just converting a single point to a small rectangle.

Reply:
oh, storing in a string a bit janky, but given that, yes, I agree.